backend/components/unread-counter/src/main/java/co/airy/core/unread_counter/Stores.java

```diff
@@ -0,0 +1,112 @@
+package co.airy.core.unread_counter;
```

Comment:
This PR:
1. Moves counting messages not read by the business into its own app 
2. Extends the topology to produce more granular "message was read" metadata that can be consumed by components